tests/wpt/mozilla/tests/mozilla/htmlanchorelment-search-attribute.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<meta charset="utf-8">
+<html>
+<head>
+<title>HTMLAnchorElement search setter crashes</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<a>anchor</a>
```

Comment:
Please put this in the directory of upstream tests, add `assert_*`s that the setter works correctly, and also test all the other setters you changed.